samples/dashboards/demo/climate/test-notes.md

```diff
@@ -1 +1,28 @@
-Click on a city marker in the world map. You can also switch the temperature scale in the context menu.
+
+# Manual tests
+
+Visual test of the initial state of the application, followed the observing the effect of user interactions.
+
+## Initial state
```

Comment:
The `Initial state` is covered with the visual test, there is no need to describe it here